sharepoint_connector_config.yml

```diff
@@ -0,0 +1,42 @@
+#Configurations for the SharePoint 2016 Connector
+
+#SharePoint 2016 configuration settings
+#The client id generated while app registration on sharepoint server to access the contents
+sharepoint.client_id: dummy
+#The secret generated while app registration on sharepoint server
+sharepoint.client_secret: dummy
+#The authentication realm of a SharePoint farm
+sharepoint.realm: dummy
+#The address of the sharepoint farm. Example: http://sharepoint-host:14293/
+sharepoint.host_url: http://10.0.9.211:14682
+#Specifies the site collections whose contents the user wants to fetch and index.
+sharepoint.site_collections: 
+    - Connector
+#Elastic WorkPlace configuration settings
+#Access token for Workplace search authentication
+ws.access_token: "52b30c01abe45261fb719d76b67b6520fa429f5a1d1ad8130838adae538982f5"                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
+#Source identifier for the custom source created on the workplace search server
+ws.source_id: "61373edd2cbdfd2b7e78d5fb"
+#Workplace search server address Example: http://10.0.1.123:3002 
+ws.host_url: "http://10.0.8.215:3002/"
```

Comment:
This is technically the host for _Enterprise Search_, which should possibly imply a configuration property name of: `enterprise_search.host_url`.

Reply:
I appreciate all the suggestions, @RossAllenBell . Many of them have been incorporated in the code. Meanwhile, I'm still working on Automated tests.

Reply:
Done